### PR TITLE
Improve bracken tagging

### LINKED ISSUE #329: Rename process name of kraken2 that goes to bracken
### Description of feature

So that log will be more recognizable.

The log file can be something like:

```diff
[2b/64e5dd] Submitted process > NFCORE_TAXPROFILER:TAXPROFILER:PROFILING:KRAKEN2_KRAKEN2 (2613_se)
- [32/f0465c] Submitted process > NFCORE_TAXPROFILER:TAXPROFILER:PROFILING:KRAKEN2_KRAKEN2 (2613_se)
+ [32/f0465c] Submitted process > NFCORE_TAXPROFILER:TAXPROFILER:PROFILING:KRAKEN2_BRACKEN (2613_se)
```

or

```diff
[2b/64e5dd] Submitted process > NFCORE_TAXPROFILER:TAXPROFILER:PROFILING:KRAKEN2_KRAKEN2 (2613_se)
- [32/f0465c] Submitted process > NFCORE_TAXPROFILER:TAXPROFILER:PROFILING:KRAKEN2_KRAKEN2 (2613_se)
+ [32/f0465c] Submitted process > NFCORE_TAXPROFILER:TAXPROFILER:PROFILING:KRAKEN2_KRAKEN2 (2613_se - bracken)
```

### PR DESCRIPTION
**Pull request by jfy133**
closes #329 

image

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#304](https://github.com/nf-core/taxprofiler/pull/304) Correct mistake in kaiju2table documentation, only single rank can be supplied (♥ to @artur-matysik for reporting, fix by @jfy133)
 - [#307](https://github.com/nf-core/taxprofiler/pull/307) Fix databases being sometimes associated with the wrong tool (e.g. Kaiju) (fix by @jfy133)
 - [#313](https://github.com/nf-core/taxprofiler/pull/304) Fix pipeline not providing error when database sheet does not have a header (♥ to @noah472 for reporting, fix by @jfy133)
+- [#329](https://github.com/nf-core/taxprofiler/pull/329) Added better tagging to allow disambiguation of Kraken2 steps of Kraken2 vs Bracken (♥ to @MajoroMAsk for requesting, added by @jfy133)
```
Comment (jfy133) [inline CHANGELOG.md:35]:
```suggestion
- [#330](https://github.com/nf-core/taxprofiler/pull/330) Added better tagging to allow disambiguation of Kraken2 steps of Kraken2 vs Bracken (♥ to @MajoroMAsk for requesting, added by @jfy133)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -32,6 +32,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#304](https://github.com/nf-core/taxprofiler/pull/304) Correct mistake in kaiju2table documentation, only single rank can be supplied (♥ to @artur-matysik for reporting, fix by @jfy133)
 - [#307](https://github.com/nf-core/taxprofiler/pull/307) Fix databases being sometimes associated with the wrong tool (e.g. Kaiju) (fix by @jfy133)
 - [#313](https://github.com/nf-core/taxprofiler/pull/304) Fix pipeline not providing error when database sheet does not have a header (♥ to @noah472 for reporting, fix by @jfy133)
+- [#330](https://github.com/nf-core/taxprofiler/pull/330) Added better tagging to allow disambiguation of Kraken2 steps of Kraken2 vs Bracken (♥ to @MajoroMAsk for requesting, added by @jfy133)
```
Comment (jfy133) [inline CHANGELOG.md:35]:
```suggestion
- [#330](https://github.com/nf-core/taxprofiler/pull/330) Added better tagging to allow disambiguation of Kraken2 steps of Kraken2 vs Bracken (♥ to @MajoroMask for requesting, added by @jfy133)
```

File: lib/WorkflowTaxprofiler.groovy [hunk 3]
```diff
@@ -93,7 +93,7 @@ class WorkflowTaxprofiler {
                 params.run_bracken ? "Bracken (Lu et al. 2017)," : "",
                 params.run_kraken2 ? "Kraken2 (Wood et al. 2019)," : "",
                 params.run_krakenuniq ? "KrakenUniq (Breitwieser et al. 2018)," : "",
-                params.run_metaphlan3 ? "MetaPhlAn3 (Beghini et al. 2021)," : "",
+                params.run_metaphlan ? "MetaPhlAn3 (Beghini et al. 2021)," : "",
```
Comment (LilyAnderssonLee) [inline lib/WorkflowTaxprofiler.groovy:96]: Thanks for this correction.
`params.run_metaphlan ? "MetaPhlAn (Blanco-Miguez et al. 2023)," : "",`

Comment (jfy133) [reply lib/WorkflowTaxprofiler.groovy]: Oops, how did I miss that :face_exhaling:

**Comment by LilyAnderssonLee** [comment]
Great job on making the Kraken and Bracken processes more recognizable in the log file!!!

**Comment by jfy133** [comment]
@LilyAnderssonLee if you're happy with it, please press the green 'review' and press' approve :)

**Pull request merged**